packages/cli/src/net/createFetch.mts

```diff
@@ -73,6 +76,17 @@ function handleFetchError(e: unknown): Promise<Response> {
     message = "The site version already exists";
   } else if (e.errorName === "VersionNotFound") {
     message = "The site version could not be found";
+  } else {
+    const { errorCode, errorName, errorInstanceId, parameters } = e;
+    // Include extra info about the original API error in CLI error messages
+    // https://www.palantir.com/docs/foundry/api/general/overview/errors/
```

Comment:
I added this to the message in CLI only so it won't affect any other places that are using the message from PalantirApiError e.g. browser console which may be able to get these details elsewhere